addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1650,6 +1650,13 @@ const Wysiwyg = Widget.extend({
         if (this.options.snippets) {
             commands.push(...this._getSnippetsCommands());
         }
+        if (this.options.plugins) {
+            if ('LinkPlugin' in this.options.plugins && 'MediaPlugin' in this.options.plugins) {
```

Comment:
since there is option to block one or the other, it would probably make more sense to not group them

also it would make sense to check `this.options.plugins.LinkPlugin !== false` instead of what is currently done

and instead of filtering, it would be more straightforward to add command, eg: `if(!this.options.plugins || this.options.plugins.LinkPlugin !== false) // add Navigation commands)